### DISCUSSION TITLE
Support Stage Channel

### DISCUSSION THREAD

**Discussion by thnhmai06** [HELLSNAKES/Music-Slash-Bot#3] [Q&A] [answered]
You can make bot play music in **Stage Channel** by using [.setSuppressed()](https://discord.js.org/#/docs/main/stable/class/VoiceState?scrollTo=setSuppressed) and [.setRequestToSpeak()](https://discord.js.org/#/docs/main/stable/class/VoiceState?scrollTo=setRequestToSpeak). Let's try it @HELLSNAKES :)

**Comment by HELLSNAKES** [comment]
@thanhgaming5550 distube has supported playing in the stage channel

**Comment by HELLSNAKES** [comment, marked as answer]
@thanhgaming5550 ![unksnown](https://user-images.githubusercontent.com/63496374/141471881-2f5d471a-636f-4e12-b3ae-0d80299b8eb8.png)
